src/transformer/get_event_function_map.php

```diff
@@ -56,6 +56,7 @@ function get_event_function_map() {
         '\core\event\course_module_completion_updated' => 'core\course_module_completion_updated',
         '\mod_assign\event\assessable_submitted' => 'mod_assign\assignment_submitted',
         '\mod_assign\event\submission_graded' => 'mod_assign\assignment_graded',
+        '\mod_assign\event\course_module_viewed' => 'all\course_module_viewed',
```

Comment:
presuming this just gels perfectly with the module viewer like the others?

Reply:
yep. No one gets their own special handler anymore